src/libsync/abstractnetworkjob.h

```diff
@@ -98,6 +98,10 @@ class OWNCLOUDSYNC_EXPORT AbstractNetworkJob : public QObject
      */
     static int httpTimeout;
 
+    /** whether or noth this job can be retried */
+    bool retryAble() const;
```

Comment:
Perhaps this should be more explicit that this is only about retrying after renewing the auth token.
So the name should be something like `isRetryableAfterAuthToken`
And actually, i would change it for something different as to describe that this is actually means that the job should work when there is no authentication:  `isUnauthenticatedJob`

(as for Qt API naming convention,  boolean starts with 'is' and are named in such a way that the default is false)

Reply:
hm maybe it would be best to call it  isAuthenticationJob() as only those are not allowed to be retried?

Reply:
:+1: